hooks/pre-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo "Running linter tests"
+lines=$(sh scripts/run_linter.sh | wc -l)
+if [ $lines -eq 1 ]; then
+    echo "Linter tests passed"
+else
+    # Assign standard input to keyboard
+    exec < /dev/tty
```

Comment:
If omitted, the script does not wait for an input ([reference](http://stackoverflow.com/questions/3417896/how-do-i-prompt-the-user-from-within-a-commit-msg-hook))